infrastructure/application/index.ts

```diff
@@ -100,14 +100,6 @@ export = async () => {
     securityGroups: [
       new awsx.ec2.SecurityGroup("metabase-custom-port", {
         vpc,
-        ingress: [
-          {
-            protocol: "tcp",
-            cidrBlocks: ["0.0.0.0/0"],
-            fromPort: 443,
-            toPort: 443,
-          },
-        ],
```

Comment:
Had an [issue with a duplicate security group rule being set](https://github.com/terraform-aws-modules/terraform-aws-security-group/issues/225), which made `pulumi up` fail. I've removed this explicitly set ingress rule from here and metabase is still accessible on [metabase.planx.in](https://metabase.planx.in).

If this change ends up disrupting our production metabase service, we'll get a UptimeRobot alert and I (we) will know this is probably the culprit, so we can fix it.